### PR TITLE
Data grabbing

### PR DESCRIPTION
**Pull request by CalebCyphers**
## Description

This pull request adds Fetch Requests for data and the classes to contain them, then links those classes to index.html

## Affected areas of application

- index.js
- User.js
- TravelRepository.js

## How Has This Been Tested?

- Tested in google chrome using Live Server
- Console.logs()

## Relevant Tickets

Progress on #/2

## Screenshots

Screen Shot 2020-09-22 at 4 24 33 PM

### CODE REVIEW

File: src/TravelRepository.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class TravelRepository {
```
Comment (bretm9) [inline src/TravelRepository.js:1]: Good call making a repo for destinations and trips.
Make sure you have testing for this class!

File: src/index.js [hunk 2]
```diff
@@ -3,9 +3,20 @@
...
 import './images/turing-logo.png'
 
+let traveler = new User(10);
+let repository = new TravelRepository;
+
+traveler.getUserData();
+repository.getDestinations();
+repository.getTrips();
+
+setTimeout(function() {
```
Comment (bretm9) [inline src/index.js:17]: I'm assuming this setTimeout is just a test function for the console log. Smart way of seeing if the data is coming in! Are you planning on the rest of the functionality to live in a setTimeout like this so you have access to the data?

**Review by bretm9 — APPROVED** [review]
This is good headway. Make sure you build in Chai! Let me know if you need me on any future PRs!

File: src/index.js (hunk 2 above)
Comment (CalebCyphers) [reply src/index.js:17]: Yes, this is here for testing. You bring up an interesting point though, not sure exactly how I'll deal with that *thinking cap on*

**Pull request merged**